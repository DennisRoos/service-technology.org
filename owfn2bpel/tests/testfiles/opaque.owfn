PLACE
INTERNAL p1, p2, p3, p4, p5, p6;
INPUT a;
OUTPUT b;

INITIALMARKING p1:1;
FINALMARKING p6:1;

TRANSITION t1
CONSUME p1;
PRODUCE p2, p3;

TRANSITION t2
CONSUME p2, a;
PRODUCE p4;

TRANSITION t3
CONSUME p3;
PRODUCE p5, b;

TRANSITION t4
CONSUME p4, p5;
PRODUCE p6;
