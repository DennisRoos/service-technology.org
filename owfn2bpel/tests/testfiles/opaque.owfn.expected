<process name="Act_process_opaque.owfn"
  targetNamespace="http://docs.oasis-open.org/wsbpel/2.0/process/abstract"
  xmlns="http://docs.oasis-open.org/wsbpel/2.0/process/abstract"
  suppressJoinFailure="yes"
  xmlns:template="http://docs.oasisopen.org/wsbpel/2.0/process/abstract/simple-template/2006/08"
  abstractProcessProfile="http://docs.oasisopen.org/wsbpel/2.0/process/abstract/simple-template/2006/08">
    <partnerLinks>
      <partnerLink name="generic_pl" partnerLinkType="##opaque"
                   myRole="##opaque" partnerRole="##opaque" />
    </partnerLinks>

  <variables>
    <variable name="Var_a" element="##opaque" />
    <variable name="Var_b" element="##opaque" />
  </variables>

  <flow name="Act_t1">
    <receive name="Act_a"
      partnerLink="generic_pl"
      operation="a"
      variable="Var_a" />
    <invoke name="Act_b"
      partnerLink="generic_pl"
      operation="b"
      inputVariable="Var_b" />
  </flow>

</process>

